src/header/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { HeaderContainer } from './style';
+import { ReactComponent as LeadhomeLogo } from '../../assets/leadhome-logo.svg';
```

Comment:
It would be handy to be able to to pass a logo in so that other people can use it.

Reply:
Definitely, going to add that in

Reply:
Leaving the LH one as default so that our projects don't need the svg.